src/main/java/org/passay/AllowedCharacterRule.java

```diff
@@ -40,6 +41,17 @@ public AllowedCharacterRule(final char[] c)
   }
 
 
+  /**
+   * Create a new allowed character rule.
+   *
+   * @param  cp  allowed character code points
+   */
+  public AllowedCharacterRule(final int[] cp)
```

Comment:
The use of int is jarring here. Given the central concern of rule construction as a means to implementing password policy, I'd like to see a `CodePoint` type that facilitates conversion of strings and characters to the types needed by this API:

    new AllowedCharacterRule(new CodePoint("abcdefghikjlmnopqrstuvwxyz"));

You have `PaswordUtils` performing these conversions now, but I'd like to see all the string-to-code-point conversion concerns moved to this type.

Reply:
Fixed, added `CodePoints` class.